python/GafferSceneUI/TransformToolUI.py

```diff
@@ -107,8 +107,6 @@ def __init__( self, tool, **kw ) :
 
 		GafferUI.Frame.__init__( self, borderWidth = 4, **kw )
 
-		self._qtWidget().setObjectName( "gafferDarker" )
```

Comment:
Given the importance of this little status bar, and the fact that folks often seem to miss what it is saying, I think it is too transparent now.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/f956d7a11eb96ec38c163c06ccff6c49790d6acd